.github/workflows/backup-db.yml

```diff
@@ -78,3 +88,45 @@ jobs:
         backup-file: ${{ env.BACKUP_FILE }}.sql
         db-server-name: ${{ inputs.db-server }}
         slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
+
+    - name: Create local sanitised database
```

Comment:
I imagine we only want to run this for the scheduled backup, not when running an on-demand backup? If so add 
if: github.event_name == 'schedule'
to all the new steps

Reply:
Will do when it's ready to merge. Right now, just adding it as a comment so we it can be tested